### PR TITLE
javascript adding

### LINKED ISSUE #6: an initial file of js

### PR DESCRIPTION
**Pull request by nabilramy**
relates #6
i made the first block of the js files that initiate the list, now im uploading it to let my team start working with it

### CODE REVIEW

**Review by basillubbad — APPROVED** [review]
👍

**Review by hussambanna — APPROVED** [review]

**Review by IsraaAmani — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
 </head>
+
 <body>
+    <section class="section-todo">
+        
+            <div class="rest-todo">
+
+
+                <img class="moon-img"  src="./img/Vector.png" alt="">
+                <span id="text-rest">Reset</span>
```
Comment (sara219) [inline index.html]: there's no vector image inside the `img` folder

Screen Shot 2021-11-11 at 9 31 46 AM

File: index.html [hunk 2]
```diff
@@ -1,14 +1,55 @@
...
+
+
+            </div>
+
+            <div class="row-list">
+
+                <div class="img-list"><img src="./img/clarity_list-outline-badged.png" alt=""></div>
+                <div class="to-do-list">
```
Comment (sara219) [inline index.html]: the same here, there's no image with that name 

Screen Shot 2021-11-11 at 9 37 30 AM

**Review by sara219 — CHANGES_REQUESTED** [review]
provide some description (in issue or here in the description) that you're working on the base of the js file and test the add item task. 

you didn't give us any details about what you added. till we open the file we will know what you have done and what you're working on

**Comment by nabilramy** [comment]
i made the first block of the js files that initiate the list, now im uploading it to let my team start working with it

**Review by sara219 — APPROVED** [review]

**Pull request merged**